setup.py

```diff
@@ -61,15 +61,28 @@ def run(self):
     # Build regular proto files.
     protos = glob.glob(os.path.join(self.indir, '*.proto'))
     if protos:
-      print "Attempting to build proto files:\n%s" % '\n'.join(protos)
-      subprocess.check_call([
+      print 'Attempting to build proto files:\n%s' % '\n'.join(protos)
+      cmd = [
           self.protoc,
           '--proto_path', self.indir,
           '--proto_path', self.protodir,
           '--python_out', self.outdir,
-          ] + protos)
+      ] + protos
+      try:
+        subprocess.check_call(cmd)
+      except OSError as e:
+        if e.errno == 2:
+          print 'You need to install the protobuf compiler.'
```

Comment:
Done. ENOENT means the binary didn't exist, so protoc wasn't at the location so I added that message.